ci/docker_push.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -ev
+want_push='false'
+for branch in "master" "1.0.0" "1.1.0"
```

Comment:
Initially I thought these would work for our tagged releases but then I realized this would require us maintaining actual release branches not just having a tagged release.